### PR TITLE
 Build 3.0 AggregateSite extension 

### PR DESCRIPTION
**Pull request by pakrym**
Fixes: https://github.com/aspnet/AspNetCore/issues/4570
Part of https://github.com/aspnet/AspNetCore-Internal/issues/1658
Include 3.0 packages in it.

### CODE REVIEW

**Review by natemcmaster — APPROVED** [review]

File: src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <title>ASP.NET Core Logging Extensions</title>
-    <Description>This extension enables additional functionality for ASP.NET Core on Azure WebSites, such as enabling Azure logging.</Description>
+    <title>ASP.NET Core Logging Integration</title>
+    <Description>This site extension enables logging integration for ASP.NET Core applications on Azure App Service.</Description>
```
Comment (pakrym) [inline src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: While doing https://github.com/aspnet/AspNetCore/issues/4570 I thought that having `extension[s]` in the name is less clear then just integration.

cc @shirhatti @muratg

Comment (muratg) [reply src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: Sounds good to me.

Comment (shirhatti) [reply src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: Does this surface anywhere in the portal UI? If so, we'll need a doc bug tracking the change in the title

Comment (pakrym) [reply src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj]: https://github.com/aspnet/Docs/issues/9968

**Comment by pakrym** [comment]
https://github.com/aspnet/AspNetCore/issues/6870

**Comment by pakrym** [comment]
@muratg, Yes, that test failed in PR check. Edited the comment.

File: src/SiteExtensions/LoggingAggregate/version.props [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>3.0.0</VersionPrefix>
-    <VersionSuffix>preview</VersionSuffix>
```
Comment (muratg) [inline src/SiteExtensions/LoggingAggregate/version.props]: cc @shirhatti

Comment (pakrym) [reply src/SiteExtensions/LoggingAggregate/version.props]: Prefix update in necessary so when we start building site extension as part of 3.0 previews we won't get the conflicting versions.

**Pull request merged**